scripts/secnetperf.ps1

```diff
@@ -215,20 +215,69 @@ function CheckRegressionTput($values, $testid, $transport, $regressionJson) {
     $envStr = "$os-$arch-$environment-$io-$tls"
     $Testid = "$testid-$transport"
     try {
+        # TODO: baseline is a bad name. Use LowerThreshold / UpperThreshold instead.
         $baseline = $regressionJson.$Testid.$envStr.baseline
     } catch {
         Write-Host "No regression baseline found"
         return "NULL"
     }
+
+    try {
+        $BestResult = $regressionJson.$Testid.$envStr.BestResult
+        $Noise = $regressionJson.$Testid.$envStr.Noise
+        $BestResultCommit = $regressionJson.$Testid.$envStr.BestResultCommit
+        if ($avg -lt $baseline) {
+            Write-GHError "🤮 Regression detected in $Testid for $envStr. Baseline: $baseline, New: $avg, BestResult: $BestResult, Noise: $Noise, BestResultCommit: $BestResultCommit"
```

Comment:
For now, let's change all regression detections to warnings, so as to not block progress.

Reply:
This is mainly for printing stuff to console for observability, it doesn't block progress.

the line `run: ./.github/workflows/generate-summary.ps1 --BlockOnFailure ${{ github.event.client_payload.pr != '' && !inputs.commit }}` from https://github.com/microsoft/netperf/blob/main/.github/workflows/quic.yml is what determines if the workflow should stop (exit code 1) or not if there is a regression.